tests/test_visitors/test_ast/test_conditions/test_negated_conditions.py

```diff
@@ -135,3 +141,55 @@ def test_correctly_negated_complex_conditions(
     visitor.run()
 
     assert_errors(visitor, [])
+
+
+@pytest.mark.parametrize('template', [
+    simple_if_exp,
+])
+@pytest.mark.parametrize('code', [
```

Comment:
We have to reuse these examples. Maybe by creating a custom `@pytest.fixture()`?